src/binaryen-c.cpp

```diff
@@ -5878,6 +5878,14 @@ BinaryenType BinaryenFunctionGetVar(BinaryenFunctionRef func,
   assert(index < vars.size());
   return vars[index].getID();
 }
+BinaryenIndex BinaryenFunctionAppendVar(BinaryenFunctionRef func,
+                                        BinaryenType type) {
+  auto* function = (Function*)func;
+  auto& list = function->vars;
+  auto index = list.size();
+  list.push_back((Type)type);
```

Comment:
This can use `Builder::addVar`.

For consistency with that API, perhaps this could be `BinaryenFunctionAddVar`?